lib/jobs/secure-erase-drive.js

```diff
@@ -439,5 +440,57 @@ function secureEraseJobFactory(
         };
     };
 
+    /**
+     * @memberOf SecureEraseJob
+     * @function _classifyDiskPara
+     * @description classify disk informations inside eraseSettings into devName list and wwid list
+     */
+    SecureEraseJob.prototype._classifyDiskPara = function() {
+        var eraseSettings = this.eraseSettings;
+        _.forEach(eraseSettings, function(eraseSetting){
+            var diskList = [];
+            var diskWwid = {};
+            _.forEach(eraseSetting.disks, function(disk){
+                //To make sure rackhd drive catalog matches real situation on nodes
+                //user is asked to input both devname and esxiwwid for each disk
+                if (disk.devName) {
+                    diskList.push(disk.devName);
+                } else {
+                    throw new Error("No devName is found from user input disk list");
+                }
+
+                if (disk.esxiWwid){
+                    diskWwid[disk.devName] = disk.esxiWwid;
+                } else{
+                    throw new Error("No esxiWwid is found from user input disk list");
+                }
```

Comment:
currently there is no schema defined for secure erase job, so in our rule, this is not a hard requirement for this PR.
But if anyone can help add the missing schema, that's will be very appreciate.